.changeset/lucky-falcons-dress.md

```diff
@@ -0,0 +1,39 @@
+---
+'@backstage/cli': minor
+'@backstage/create-app': minor
```

Comment:
I believe that, because this change is not breaking from the perspective of either of these packages, and because both of these packages are still `0.x`, the appropriate change level is `patch` in this case.